examples/ev-chargers.py

```
@@ -0,0 +1,138 @@
+        else:
+            result = await gb.ev_stations_nearby(
+                lat=args.lat,
+                lon=args.lon,
+                radius=args.radius,
+                networks=args.networks,
+                connector_types=args.connectors,
+                charging_levels=args.levels,
+                limit=args.limit,
+            )
+    except (LibraryError, APIError) as e:
+        raise SystemExit(f"Error fetching EV stations: {e}")
```

Ruff (0.15.12)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)